doc/advancedFeatures.rst

```diff
@@ -0,0 +1,43 @@
+Advanced Features
+=================
+.. Gradient Evaluation with Complex Step
+.. -------------------------------------
+
+.. Parallel Execution
+.. ------------------
+
+.. Storing Optimization History
+.. ----------------------------
+
+Hot start
+---------
+Hot start refers to the way optimizations are initialized.
```

Comment:
I can add a one-liner about the difference with "cold" starts?

Reply:
Yep good idea.